exporter/collector/testdata/fixtures/workload_metrics_expect.json

```diff
@@ -6452,7 +6467,7 @@
     },
     {
       "name": "googlecloudmonitoring/point_count",
-      "val": "200",
```

Comment:
There are three points in this fixture, each with no percentiles. So this increased by three because each one contributes sum and count now.